addons/web_tour/static/tests/tour_manager_tests.js

```diff
@@ -71,12 +71,48 @@ odoo.define('web_tour.tour_manager_tests', async function (require) {
                 debug: true,
             });
 
-            assert.containsOnce(document.body, '.o_tooltip');
-            const tooltip = document.querySelector('.o_tooltip_content');
-            assert.strictEqual(tooltip.innerHTML.trim(), "Oui",
+            assert.containsOnce(document.body, '.o_tooltip:visible');
+            assert.strictEqual($('.o_tooltip_content:visible').text(), "Oui",
```

Comment:
Dropping this here because it's been on my mind for a while: what is our current stand on jQuery (or libs in general) ? Do we want to remove it from the framework but still use it in tests ? It has been a while since the last discussion on the matter of libraries and I see uses of jQuery and underscore pop a bit more often as of late

Reply:
Giving r+ nevertheless since it's not a problem related to this PR (no other problem spotted)

Reply:
I have no problem using jquery in tests. It's often quite convenient (here for the :visible modifier). There is probably no chance we'll ever remove the lib from the tests bundle.